pages/common/ln.md

```diff
@@ -4,16 +4,16 @@
 
 - create a symbolic link to a file or folder
 
-`ln -s {{path/to/original/file}} {{path/to/link}}`
+`ln -s {{path/to/target}} {{path/to/link}}`
 
 - overwrite a symbolic link to a file
 
-`ln -sf {{path/to/new/original/file}} {{path/to/file/link}}`
+`ln -sf {{path/to/new/target/file}} {{path/to/file/link}}`
 
 - overwrite a symbolic link to a folder
 
-`ln -sfT {{path/to/new/original/file}} {{path/to/folder/link}}`
+`ln -sfT {{path/to/new/target/folder/}} {{path/to/folder/link}}`
```

Comment:
The only case I can think of is something thinking

>  I need to remove this symlink.... I'll do an `rm` followed by a new `ln`.... wait, will `rm` remove the symlink or the actual file?

But then again, they'd be asking themselves the same question if they wanted to just remove the symlink (not just overwrite).